var/spack/repos/builtin/packages/synapsetool/package.py

```diff
@@ -34,6 +34,8 @@ class Synapsetool(CMakePackage):
     git      = "ssh://bbpcode.epfl.ch/hpc/synapse-tool"
 
     version('develop', submodules=True)
+    version('0.5.8', tag='v0.5.8', submodules=True)
+    version('0.5.7', tag='v0.5.7', submodules=True)
```

Comment:
If `0.5.7` is not published/released, can it be dropped?

Reply:
I believe so...